From 9257b56ca659fb9ace98d3b4e842d85b80e0efd7 Mon Sep 17 00:00:00 2001
From: Dale Whinham <daleyo@gmail.com>
Date: Sat, 29 Nov 2025 23:19:08 +0000
Subject: [PATCH] HACK: Workaround suspend/resume issues on SP11

---
 drivers/platform/surface/aggregator/controller.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/platform/surface/aggregator/controller.c b/drivers/platform/surface/aggregator/controller.c
index a265e667538c1c..3f57754853a893 100644
--- a/drivers/platform/surface/aggregator/controller.c
+++ b/drivers/platform/surface/aggregator/controller.c
@@ -1155,7 +1155,7 @@ static int ssam_controller_caps_load_from_of(struct device *dev, struct ssam_con
 	 * Every device starting with Surface Pro X through Laptop 7 uses these
 	 * identical values, which makes them good defaults.
 	 */
-	caps->d3_closes_handle = true;
+	caps->d3_closes_handle = false;
 	caps->screen_on_sleep_idle_timeout = 5000;
 	caps->screen_off_sleep_idle_timeout = 30;
 	caps->ssh_buffer_size = 48;
